### PR TITLE
[CHORE] Update react-intl package, to support html formatted messages.

### PR DESCRIPTION
**Pull request by sanderbrugge**
### Changed
- Updated the react-intl package, following their guide.

### MC
- link the package locally to another project
  - see if the existing i18n plays nice
  - see if you can add [formatted messages](https://github.com/formatjs/react-intl/blob/master/examples/Messages.tsx#L66) to the translation

### CODE REVIEW

**Review by lowiebenoot — APPROVED** [review]
Manual checked it, works fine 👍 

@ team: what version number should we use for releasing this? Go along with a react-intl and also do a major bump? So `4.0.0`?

**Review by EOSullivanBerlin — APPROVED** [review]

**Review by ArnaudWeyts — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -201,28 +142,30 @@ class Provider extends React.PureComponent<Props, State> {
...
-
-    const module = await import(`react-intl/locale-data/${language}`);
-    const localeData = module.default;
-
-    if (locale === 'tlh-KL') {
-      localeData.push({
-        locale: 'tlh',
-        parentLocale: 'en',
-      });
+  async setLocalData(locale: string): Promise<Array<Object>> {
```
Comment (ArnaudWeyts) [inline src/index.js]: Minor typo here `setLocaleData` instead of `setLocalData` 🙈

Comment (sanderbrugge) [reply src/index.js]: 😡 _why am I like this_

Comment (lowiebenoot) [reply src/index.js]: Ah yeah, lol, yesterday you actually said you made that typo 😂

Comment (sanderbrugge) [reply src/index.js]: hence my angry face 😄 it's fixed now though, but approves don't appear to get reset so 👍

**Pull request merged**